### ISSUE TITLE
Python api: training takes a lot of disk space

### ISSUE THREAD

**Issue by Erlemar** [catalyst-team/catalyst#409]
I tried to train a multilabel classification model with densenet169 as a backbone on Kaggle Kernels.

![example](https://i.imgur.com/jreQh5u.png)

Within 1 epoch all 5GB of disk space were consumed.

**Comment by TezRomacH** [comment]
Hi! Please check how many checkpoints you save (save_n_best parameter in CheckpointCallback).

**Comment by Erlemar** [comment]
I used the default value - it should be equal to 1.

**Comment by Arquestro** [comment]
Hello! Please, can you measure size of the delta between start and end of training? Is it near 5GB or smaller?

**Comment by sparkfax** [comment]
I have similar problem in kaggle
Disk space error: Disk writes might have failed due to insufficient space.
callbacks=[
        # Each criterion is calculated separately.

        CriterionCallback(
            input_key="mask",
            prefix="loss_dice",
            criterion_key="dice"
        ),
        CriterionCallback(
            input_key="mask",
            prefix="loss_bce",
            criterion_key="bce",
            multiplier=0.8
        ),      
        # And only then we aggregate everything into one loss.
        CriterionAggregatorCallback(
            prefix="loss",
            loss_keys=["loss_dice",  "loss_bce"],
            loss_aggregate_fn="sum" # or "mean"
        ),   
        # metrics
        DiceCallback(input_key="mask"),
    ]
# we have multiple criterions
criterion = {
    "dice": DiceLoss(),
    "bce": nn.BCEWithLogitsLoss()
}
num_epochs = 1
logdir = "./logs/segmentation"
runner.train(
    model=model,
    criterion=criterion,
    optimizer=optimizer,
    scheduler=scheduler,
    loaders=loaders,
    callbacks=callbacks,
    logdir=logdir,
    num_epochs=num_epochs,
    verbose=True
)


Disk 4.9G before runnning
I am thinking if there are too much output in logdir.  Is there any way to reduce output?
I did not add oupout code in my code.  I have ruduce to num_epochs = 1

/opt/conda/lib/python3.6/site-packages/catalyst/utils/checkpoint.py in save_checkpoint(checkpoint, logdir, suffix, is_best, is_last, special_suffix)
     57     torch.save(checkpoint, filename)
     58     if is_best:
---> 59         shutil.copyfile(filename, f"{logdir}/best{special_suffix}.pth")

./logs/segmentation/checkpoints/
['train.0_full.pth', 'train.0.exception_OSError.pth', 'best_full.pth']
train.0_full.pth 2.5G
best_full.pth 2.3G
